lnwallet/channel.go

```diff
@@ -5969,15 +5969,43 @@ func (lc *LightningChannel) availableBalance() (lnwire.MilliSatoshi, int64) {
 	htlcView := lc.fetchHTLCView(remoteACKedIndex,
 		lc.localUpdateLog.logIndex)
 
-	// Then compute our current balance for that view.
-	ourBalance, _, commitWeight, filteredView :=
-		lc.computeView(htlcView, false, false)
+	// Calculate our available balance from our local commitment.
+	// NOTE: this is not always accurate, since the remote node can always
```

Comment:
Style nit: should have a newline before the note section, and should start with a capital letter.

Reply:
Fixed: https://github.com/lightningnetwork/lnd/pull/3691/commits/6389d2968d636c82d630e7f38e8ef602002acd49